sw/device/tests/closed_source/meson.build

```diff
@@ -0,0 +1,37 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# All build targets below in `sw_tests` or `sw_signed_tests` will have names
+# starting with `sw/device/tests/<test_name>`. They will not contain the
+# subdirectory name (e.g., `sim_dv/`), because the build targets are really
+# declared in the for loops in this build file.
+
```

Comment:
You can remove this. This comment is not applicable in this file (only the file from the directory above, i.e., `sw/device/tests/meson.build`). Moreover, I just realized this comment is outdated in the  parent directory (where it was copied from), so thanks for pointing that out :)

Reply:
Moreover, maybe you can replace the comment you remove above with one stating that the test targets defined in this build file are only designed to build in the closed-source environement, i.e., when `partner` and `partnerdir` options are set?